src/security/rsa.cpp

```diff
@@ -11,14 +11,14 @@
 
 #include "security/rsa.h"
 
-RSA::RSA() {
+NetowrkMessageRSA::NetowrkMessageRSA() {
```

Comment:
Because the JWT library imports RSA from `openssl` we can't re-define this here, so I had to rename it.